insightsengineering/teal.logger
# Fix log shiny inputs@main

This fixes:
- logger observer doesn't trigger when anything changes (observe -> observeEvent)
- logger observer is set only if log_level is set TRACE or higher
- don't log `NULL -> NULL` anymore.

follow up:
- https://github.com/insightsengineering/teal.transform/pull/215


To test:
set `options(teal.log_level = "TRACE")` before attaching packages and run teal.gallery apps to see the difference between `main` and `{this}` branch
